src/frontend/mame/ui/dirmenu.cpp

```diff
@@ -42,6 +42,7 @@ struct folders_entry
 const folders_entry f_folders[] =
 {
 	{ N_p("path-option", "ROMs"),               OPTION_MEDIAPATH,          ADDING },
+	{ N_p("path-option", "Software Lists"),     OPTION_HASHPATH,           CHANGE },
```

Comment:
I’m pretty sure MAME supports multiple paths for this so it should be `ADDING` rather than `CHANGE`.  However this change requires regenerating localisations, etc. – it’s too close to release for that.  I’ll look at adding this next time I do something that affects the message catalogs.